distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/restito/ChunkedContent.java

```diff
@@ -196,8 +198,9 @@ private ChunkedContentFunction(
       this.headerCapture = headerCapture;
     }
 
+    @SuppressWarnings({"squid:S2629"})
```

Comment:
❓  why did you need the suppression? I assume it is for one of the logging statements in here?

Reply:
Yes. For logging.

Reply:
✏️ Can you add the reason next to the squid id?